README.md

```diff
@@ -56,15 +56,10 @@ deprecatedrpc=signrawtransaction
 
 The released binaries can be downloaded [here](https://github.com/ACINQ/eclair/releases).
 
-#### Windows
+#### Installing Java
 
-Just use the windows installer, it should create a shortcut on your desktop.
-
-#### Linux, macOS or manual install on Windows
-
-You need to first install java, more precisely a [JRE 1.8](http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html).
-
- :warning: If you are using the OpenJDK JRE, you will need to build OpenJFX yourself, or run the application in headless mode (see below).
+You need to first install java. Eclair will run on [Oracle JRE 1.8](http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html), [Oracle JDK 11](https://www.oracle.com/technetwork/java/javase/downloads/jdk11-downloads-5066655.html) and [Open JDK 11](https://jdk.java.net/11/).
```

Comment:
Let's be more clear: 

"You need to install Open JDK 11".... and then.... "Eclair will also run on X Y Z but this is not recommended.